Mathlib/NumberTheory/ModularForms/Petersson.lean

```diff
@@ -62,16 +62,82 @@ lemma petersson_slash_SL (k : ℤ) (f f' : ℍ → ℂ) (g : SL(2, ℤ)) (τ : 
   simp [σ, ModularForm.SL_slash, petersson_slash,
     -Matrix.SpecialLinearGroup.map_apply_coe, -Matrix.SpecialLinearGroup.coe_matrix_coe]
 
+lemma petersson_symm (k : ℤ) (f f' : ℍ → ℂ) (τ : ℍ) :
+    petersson k f' f τ = conj (petersson k f f' τ) := by
```

Comment:
What do you think of also adding the swapped lemma `conj_petersson` as a simp-lemma?